fb_memcache: change fb internal memcache location

c00w opened the pull request:
Summary: The fb_memcache injections location and path is changing.

Test Plan: This shouldn't be capable of failing any tests on github.

Differential Revision: D57516213


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

oulgen reviewed (approved):
OSS parts of the code looks good to me, I left additional feedback on the Fbcode side.

c00w committed:
fb_memcache: change fb internal memcache location (#126603)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/126603

The fb_memcache injections location and path is changing.

Test Plan: This shouldn't be capable of failing any tests on github.

Reviewed By: bertmaher

Differential Revision: D57516213

DanilBaibak commented:
@pytorchbot merge
